regression/Makefile

```diff
@@ -82,18 +83,31 @@ $(INITRAMFS)/benchmark:
 $(INITRAMFS_EMPTY_DIRS):
 	@mkdir -p $@
 
-$(INITRAMFS)/opt/syscall_test:
+.PHONY: $(SYSCALL_TEST_DIR)
+$(SYSCALL_TEST_DIR):
 	@make --no-print-directory -C syscall_test
 
+.PHONY: $(INITRAMFS_IMAGE)
+$(INITRAMFS_IMAGE): $(INITRAMFS_FILELIST)
+	@if ! cmp -s $(INITRAMFS_FILELIST) $(INITRAMFS_FILELIST).previous ; then \
+		echo "Generating the initramfs image..."; \
+		cp -f $(INITRAMFS_FILELIST) $(INITRAMFS_FILELIST).previous; \
+		( \
+			cd $(INITRAMFS); \
+			cut -d " " -f 2- $(INITRAMFS_FILELIST) | \
```

Comment:
Add some comments here to explain that we need to exclude the timestamp.

Reply:
~~How can I add comments when the trailing `\` always extends the comments to the next line?~~

~~I've been trying to do this, but haven't found a way yet.~~

This is a bug in Vim's highlighting of comments. Comments can be added here without problems.